scripts/src/java/org/oppia/android/scripts/ci/LicenseTextsCheck.kt

```diff
@@ -0,0 +1,40 @@
+package org.oppia.android.scripts.ci
+
+import java.io.File
+
+private const val WARNING_COMMENT =
+  "<!-- Do not edit this file. It is generated by running GenerateLicenseTexts.kt. -->"
+
+/**
+ * Script to verify that the actual license texts are never checked into our VCS. It mainly
+ * verifies that the script-generated version of third_party_dependencies.xml file is never
+ * checked in.
+ *
+ * Usage:
+ *   bazel run //scripts:maven_dependencies_list_check -- <path_to_third_party_deps_xml>
+ *
+ * @param [args]: Array of [String] containg different paths required by the script
+ * - path_to_third_party_deps_xml: path to the third_party_dependencies.xml
+ *
+ * Example:
+ *   bazel run //scripts:maven_dependencies_list_check -- $(pwd)/app/src/main/res/values/third_party_dependencies.xml
+ */
+fun main(args: Array<String>) {
+  if (args.size < 1) {
+    throw Exception("Too few Arguments passed")
+  }
+  val pathToThirdPartyDepsXml = args[0]
+  val thirdPartyDepsXml = File(pathToThirdPartyDepsXml)
+  val xmlContent = thirdPartyDepsXml.readLines().joinToString(separator = "\n")
+
+  checkIfCommentIsPresent(xmlContent = xmlContent, comment = WARNING_COMMENT)
+
+  println("License texts Check Passed")
+}
+
+private fun checkIfCommentIsPresent(xmlContent: String, comment: String) {
+  if (!xmlContent.contains(comment)) {
+    println("Please revert the changes in third_party_dependencies.xml")
+    throw Exception("License texts checked into VCS")
```

Comment:
```suggestion
    throw Exception("License texts potentially checked into VCS")
```

Technically it's not guaranteed since you aren't verifying it (someone might have just removed or changed the warning, for example).

Reply:
Done.